### PR TITLE
Update demonet references; refactor formatting

### PR DESCRIPTION
**Pull request by orkunkl**
Related to https://github.com/CosmWasm/testnets/issues/2

### CODE REVIEW

File: getting-started/using-the-sdk.md [hunk 1]
```diff
@@ -91,16 +90,16 @@ Using the REPL:
 
 ```js
 // Create or load account
-const mnemonic = loadOrCreateMnemonic("fred.key");
-mnemonicToAddress("cosmos", mnemonic)
+const mnemonic = loadOrCreateMnemonic('fred.key')
```
Comment (orkunkl) [inline getting-started/using-the-sdk.md:93]: These are auto-formatting applied by vscode. I was lazy to revert them, and they do no harm.

File: testnets/testnets.md [hunk 2]
```diff
@@ -17,15 +17,15 @@ Let's start rolling your node and start producing blocks in live testnet environ
 
 First of all make sure you followed the installation steps in [build requirements section](./build-requirements.md). You should have the required binaries. If you just want to copy and execute the scripts below, make sure to set up environment variables:
 
-Below is the [demonet configuration](https://github.com/CosmWasm/testnets/blob/master/demo-09/config).
+Below is the [demonet configuration](https://github.com/CosmWasm/testnets/blob/master/demo-10/config).
```
Comment (ethanfrey) [inline testnets/testnets.md:21]: Does this file exist? I don't see that directory added to the repo?

File: testnets/testnets.md [hunk 3]
```diff
@@ -17,15 +17,15 @@ Let's start rolling your node and start producing blocks in live testnet environ
...
 ```sh
 export CHAIN_ID=testing
-export TESTNET_NAME=demo-09
-export RPC=https://rpc.demo-09.cosmwasm.com:443
-export FAUCET=https://faucet.demo-09.cosmwasm.com
-export SEED_NODE=26c9c79dc62b5ddc753bb9fcce022fcc98b5a8cf@p2p.demo-09.cosmwasm.com:26656
+export TESTNET_NAME=demo-10
+export RPC=https://rpc.demo-10.cosmwasm.com:443
+export FAUCET=https://faucet.demo-10.cosmwasm.com
+export SEED_NODE=<WIP>@p2p.demo-10.cosmwasm.com:26656
```
Comment (ethanfrey) [inline testnets/testnets.md]: Just query your rpc server to get this

**Review by ethanfrey — APPROVED** [review]
Nice work, please make sure all referenced config files are checked in. Otherwise, good to go. And thanks for updating the demo instructions. We will have to update rust code for 0.10 as well

File: testnets/testnets.md (hunk 2 above)
Comment (orkunkl) [reply testnets/testnets.md:21]: https://github.com/CosmWasm/testnets/pull/5

File: testnets/testnets.md [hunk 4]
```diff
@@ -131,7 +132,7 @@ Now you should be seeing blocks being replayed and your node is catching up with
 
 If you want to participate in active block building, you need some coins staked to your validators address. If you are interested in validator tech stack, [certus one blog](https://kb.certus.one/) is a good resource to begin with.
 
-**Note: make sure your validator is synced before upgrading to validator
+\*\*Note: make sure your validator is synced before upgrading to validator
```
Comment (orkunkl) [inline testnets/testnets.md]:
```suggestion
**Note**: make sure your validator is synced before upgrading to validator
```

**Pull request merged**